net/dnscrypt-proxy2/Makefile

```diff
@@ -38,6 +39,21 @@ define Package/dnscrypt-proxy2
   CONFLICTS:=dnscrypt-proxy
 endef
 
+define Package/$(PKG_NAME)/config
+config $(PKG_NAME)_INCLUDE_GOPROXY
```

Comment:
Please do not use these confusing $(PKG_NAME) variables in symbol and sector declarations